site/content/en/docs/tasks/integrate_a_custom_job.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Integrate a custom Job"
+date: 2023-07-25
+weight: 8
+description: >
+  Integrate a custom Job with Kueue.
+---
+# Integrate a custom Job with Kueue
+
+Kueue integrates with a couple of jobs, including batch Job, MPI Job, Ray Job or JobSet. 
+If there is another Job, which you would like supported, please consider contributing. 
+This page shows how to integrate a custom Job with Kueue.
+
+## What you need to specify
+
+1. Configuration
+2. Controller
+3. Webhook
+4. Adjust build system
+5. Test files
+
+## Configure your custom Job
```

Comment:
Maybe better to make it a `###Configure your custom Job` and make the name consist with the names under `What you need to specify`.